ui/components/app/signature-request-original/signature-request-original.component.js

```diff
@@ -191,37 +189,17 @@ export default class SignatureRequestOriginal extends Component {
   onSubmit = async (event) => {
     const { clearConfirmTransaction, history, mostRecentOverviewPage, sign } =
       this.props;
-    const { trackEvent, type } = this.context;
 
     await sign(event);
-    trackEvent({
-      category: EVENT.CATEGORIES.TRANSACTIONS,
-      event: 'Confirm',
-      properties: {
-        action: 'Sign Request',
-        legacy_event: true,
-        type,
-      },
-    });
     clearConfirmTransaction();
     history.push(mostRecentOverviewPage);
   };
 
   onCancel = async (event) => {
     const { clearConfirmTransaction, history, mostRecentOverviewPage, cancel } =
       this.props;
-    const { trackEvent, type } = this.context;
 
     await cancel(event);
-    trackEvent({
```

Comment:
I might be a little confused here, but I thought this cancel event would still be valid to track. Maybe we could keep this trackEvent and distinguish this cancel event from the warning modal?